asyncapi/twilio/data_types.bal

```diff
@@ -14,6 +14,24 @@
 // specific language governing permissions and limitations
 // under the License.
 
+# Listener Configuration.
+#
+# + accountSId - Twilio account SID  
+# + authToken - The authentication token of the account 
+# + phoneNumberSid - The Twilio-provided string that uniquely identifies the phone number
+# + callbackURL - Callback service URL for webhook
+@display {label: "Listener Config"}
+public type ListenerConfig record {
+    @display {label: "Account SID", "description": "Twilio account SID"}
+    string accountSId;
+    @display {label: "Auth Token", "description": "The authentication token of the account"}
+    string authToken;
+    @display {label: "Phone Number Sid", "description": "The Twilio-provided string that uniquely identifies the phone number"}
+    string phoneNumberSid;
+    @display {label: "Call Back URL", "description": "Callback service URL for webhook"}
```

Comment:
Can we make Call Back one word?

Reply:
Fixed in https://github.com/ballerina-platform/asyncapi-triggers/commit/475116df52f839b12941b4cdcc5d908a08f382f0